src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -114,6 +112,47 @@ using Psychrometrics::PsyHFnTdbW;
 using Psychrometrics::PsyRhoAirFnPbTdbW;
 using Psychrometrics::PsyTdbFnHW;
 
+enum class AirflowSpec
```

Comment:
Like having these in the `.cc` file if they are only used here!

Reply:
Yep, I 100% agree.  I'd like to script something up to check for enums only used in one cc file as well as not in any unit test files.  Although previously we wrote a script to identify all the functions in the headers that weren't used anywhere else to clean them up, and that was rejected.  Maybe I shouldn't spend time on it?